client/src/components/main/EditPassword.jsx

```diff
@@ -1,59 +1,111 @@
   }
-
   return (
-    <div onClick={onClose} className='editOverlay'>
-      <div onClick={(e) => {e.stopPropagation();}} className='editModalContainer'>
-          <div className='editDataText'>
-            Edit {item} Data
+    <div onClick={onClose} className="editOverlay">
+      <div
+        onClick={(e) => {
+          e.stopPropagation();
+        }}
+        className="editModalContainer"
+      >
+        <div className="editDataText">{`Edit ${item.websitename}`}</div>
+        <div className="websiteNameContainer">
+          <div>
+            <label htmlFor="" className="websiteName">
+              Website Name
+            </label>
           </div>
-          <div className='websiteNameContainer'>
-            <div>
-              <label htmlFor="" className='websiteName'>Website Name</label>
-            </div>
-            <div>
-                <input className='inputField' type="text" value={websiteName} onChange={(e)=>setWebsiteName(e.target.value)} placeholder='Google' />
-            </div>  
+          <div>
+            <input
+              className="inputField"
+              type="text"
+              value={websiteName}
+              onChange={(e) => setWebsiteName(e.target.value)}
+              placeholder="Google"
+            />
           </div>
-          <div className='passwordContainer'>
-            <div>
-              <label htmlFor="" className='websiteName'>Password</label>
-            </div>
-            <div>
-              <input className='inputField' type="password" value={password} onChange={(e)=>setPassword(e.target.value)} placeholder='1234567sdfgh' />
-            </div> 
+        </div>
+        <div className="passwordContainer">
+          <div>
+            <label htmlFor="" className="websiteName">
+              Password
+            </label>
           </div>
-          <div className='editBtnContainer'>
-            <button className='cancelBtn' onClick={onClose}>cancel </button>
-            <button className='saveBtn' onClick={updateFunc}>
-              <img className='saveBtnImg' src={SaveBtn} alt="" />
-              <div className='saveText'>Save</div>
+          <div className="passwordInput">
+            <input
+              className="inputField"
+              type={passwordType}
+              value={password}
+              onChange={(e) => setPassword(e.target.value)}
+              placeholder="1234567sdfgh"
+            />
+            <button
+              onClick={() => decrypt(item._id)}
+              className="eyeButton"
+            >
+              {isEyeBtnClicked ? (
+                <img src={eye} alt=""/>
+              ) : (
+                <img src={cutEye} alt=""/>
+              )}
             </button>
           </div>
-          <div className='editMessage'>
-            {message}
-          </div>
+        </div>
+        <div className="editBtnContainer">
+          <button className="cancelBtn" onClick={onClose}>
+            cancel{" "}
```

Comment:
Why cancel{" "} this not cancel only if you want to give space then do using css